build/flutter_app.gni

```diff
@@ -72,6 +72,7 @@ template("flutter_app") {
   flutter_tools_name = get_label_info(flutter_tools_label, "name")
   flutter_tools_packages = "$flutter_tools_gen_dir/$flutter_tools_name.packages"
   flutter_tools_main = "$flutter_tools_label/bin/fuchsia_builder.dart"
+  flutter_tools_asset_manifest = "$flutter_tools_label/schema/material_fonts.yaml"
```

Comment:
This file isn't the correct default.  Instead, we need to construct a default `pubspec.yaml` and use that as the default.